scripts/format-commit.sh

```diff
@@ -131,8 +131,15 @@ list_changed_files () {
 
 run_clang_format () {
 	while read -r src_file ; do
-		prepare_clang_params "$src_file" \
-			| xargs --no-run-if-empty --verbose clang-format -i
+		local ranges=()
+		while IFS=$'\n' read -r range ; do
+			ranges+=("$range")
+		done < <(git_diff_to_clang_line_range "$src_file")
```

Comment:
Ideally we could use this syntax instead of the while-loop:

`local ranges=( $(git_diff_to_clang_line_range "$src_file") )`

However, shell-check flags this as a risk: 
 - https://github.com/koalaman/shellcheck/issues/1496

Bash 4 allows this more direct (and safe) syntax: 
`readarray -t ranges < <(git_diff_to_clang_line_range "$src_file")`

And Bash 3 requires `read`ing line-by-line in a while-loop.